src/mentors.json

```diff
@@ -7709,30 +7709,62 @@
   },
   {
     "id": "jordanmccann64@outlook.com",
-    "name": "Jordan McCann",                          
-    "avatar": "https://avatars2.githubusercontent.com/u/11492379?s=400&u=5d896133c4c7459aef3388893e40e86d4fbccb73&v=4",  
-    "title": "Game Developer",                  
-    "description": "Passionate Software & Game Developer currently finishing his degree.",    
-    "country": "GB",                              
-    "spokenLanguages": [                          
+    "name": "Jordan McCann",
+    "avatar": "https://avatars2.githubusercontent.com/u/11492379?s=400&u=5d896133c4c7459aef3388893e40e86d4fbccb73&v=4",
+    "title": "Game Developer",
+    "description": "Passionate Software & Game Developer currently finishing his degree.",
+    "country": "GB",
+    "spokenLanguages": [
       "en"
     ],
-    "tags": [                                     
-      "c#", "c++", "nodejs"            
+    "tags": [
+      "c#",
+      "c++",
+      "nodejs"
     ],
-    "channels": [                                 
+    "channels": [
       {
-        "type": "email",                          
-        "id": "jordanmccann64@outlook.com"                    
+        "type": "email",
+        "id": "jordanmccann64@outlook.com"
       },
       {
-        "type": "twitter",                          
-        "id": "InfiniteShockz"                    
+        "type": "twitter",
+        "id": "InfiniteShockz"
       },
       {
         "type": "linkedin",
         "id": "jordan-mccann-b808b4156"
       }
     ]
+  },
+  {
+    "id": "arshinde123.2013@gmail.com",
+    "name": "Arvind Shinde",
+    "avatar": "https://firebasestorage.googleapis.com/v0/b/arvindshinderesume.appspot.com/o/a_DSC_0034.jpg?alt=media&token=caf2519d-2682-4703-ae34-23187aaf2766",
+    "title": "Full Stack Developer",
+    "description": "Always eager to learn and share",
+    "country": "IN",
+    "spokenLanguages": [
+      "en",
+      "hi",
+      "mr"
+    ],
+    "tags": [
+      "nodejs react express"
```

Comment:
Each tag should be a separated item in the array.
Also, notice the tag's synonyms: https://github.com/Coding-Coach/find-a-mentor#synonyms-tags